src/classes/STG_PanelRD_CTRL.cls

```diff
@@ -177,4 +189,36 @@ public with sharing class STG_PanelRD_CTRL extends STG_Panel {
         private set;
     }
 
+    public List<SelectOption> listInstallmentCreationOptions {
+        get {
+        	if (listInstallmentCreationOptions == null) {
+	            listInstallmentCreationOptions = getCreationOptions();
+        	}
+            return listInstallmentCreationOptions;
+        }
+        private set;
+    }
+
+    public String installmentAutoCreationLabel {
+        get {
+            installmentAutoCreationLabel = installmentAutoCreationLabelMap.get(STG_Panel.stgService.stgRD.Installment_Opportunity_Auto_Creation__c);
+            return installmentAutoCreationLabel;
+        }
+        private set;
+    }
+
+    private List<SelectOption> getCreationOptions() {
+        List<SelectOption> options = new List<SelectOption>();
+
+        for (String creationOption : installmentAutoCreationLabelMap.keySet()) {
```

Comment:
Can we bypass the map creation because installment Opp creation options are relevant only for the `SelectOption` list?